ui/src/message_popup/renderers/minimal_timed_view.jsx

```diff
@@ -0,0 +1,68 @@
+/* @flow weak */
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+
+// This response supports timed auto advance to next slide
+export default React.createClass({
+  displayName: 'MinimalTimedView',
+
+  propTypes: {
+    onLogMessage: React.PropTypes.func.isRequired,
+    onResponseSubmitted: React.PropTypes.func.isRequired,
+    recordText: React.PropTypes.string,
+  },
+
+  getDefaultProps() {
+    return {
+      recordText: 'Respond',
+      autoFocus: true,
+      textHeight: 48
+    };
+  },
+
+
+  componentDidMount() {
+    this.sceneTimer = setTimeout(this.onTimer, 25000);
+  },
+
+  componentWillUnmount(){
+    clearTimeout(this.sceneTimer);
+  },
+
+  sceneTimer: (null: ?number),
+  
+
+  onTimer() {
+    this.props.onLogMessage('timer_auto_advance_response');
+    this.props.onResponseSubmitted({ignore: true});
+  },
+
+
+  onSkipTapped() {
+    this.props.onLogMessage('timer_auto_advance_response', {skipTapped: true});
+    this.props.onResponseSubmitted({ignore: true});
+  },
+
+
+  render() {
+    const {
+      recordText
+    } = this.props;
+   
+    return (
+      <div className="MinimalTimedView" style={styles.container}>
+        <div style={styles.buttonRow}>
```

Comment:
This is referencing `styles.buttonRow`, but looks like that's been deleted?  If this looks the way you want visually on the screen, then you can just remove the `style={...}` bit.